_posts/2015-09-23-rolemodels.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title:  "#rolemodels Edition Six"
+date:   2015-09-23
+venue: "SoundCloud HQ, Rheinsberger Str. 76/77, Berlin 10115"
+ticket: "RSVP"
+time: "06:30pm"
```

Comment:
"06:30pm" is non-sense: the leading 0 implies a 24 hour clock. Should be either times is either 18:00 or 6:30pm. Are all the times in this non-sense format?